src/sources/multisrc/wqmangastream/WQMangaStreamScraper.js

```diff
@@ -8,6 +8,7 @@ class WQMangaStreamScraper {
     this.sourceName = sourceName;
     this.language = options?.language;
     this.reverseChapters = options?.reverseChapters;
+    this.AI_Text = options?.AI_Text;
```

Comment:
To be generic this should be something like `ignoredStrings` which would be an array of strings to filter out.